### PR TITLE
build01: add initial nixpkgs-update configuration

### PR DESCRIPTION
**Pull request by ryantm**

### CODE REVIEW

File: build01/nixpkgs-update.nix [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+let
+  userLib = import ../users/lib.nix { inherit lib; };
+
+  nixpkgs-update-src = pkgs.fetchFromGitHub {
+    owner = "ryantm";
+    repo = "nixpkgs-update";
+    rev = "02e6ccfd26572269e23dc46df615ee48aec470ca";
+    sha256 = "0rw1rzd9x3b4r6xjr2m7hd3xmyji26znn6lk83x1m1fnds10b6jr";
+  };
+  nixpkgs-update = import nixpkgs-update-src { };
```
Comment (zimbatm) [inline build01/nixpkgs-update.nix]: you can use `niv add ryantm/nixpkgs-update` and then here

```suggestion
  sources = import ../nix/sources.nix; 
  nixpkgs-update = import sources.nixpkgs-update { };
```

Comment (ryantm) [reply build01/nixpkgs-update.nix]: Okay I used niv.

**Comment by ryantm** [comment]
> looks good, do you want me to deploy it?

Am I okay to deploy it, or should I let you do it generally?

**Comment by zimbatm** [comment]
As long as we don't step onto each-other it's fine.

Can you send me your GPG public key so I can add you to git-crypt?

**Comment by ryantm** [comment]
@zimbatm @adisbladis already added me to git-crypt :)

**Pull request merged**